<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upload_674</class>
 <widget class="QMainWindow" name="Upload_674">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload_674</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>566</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblHost">
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHost">
       <property name="text">
        <string>w0152abe.kasserver.com</string>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUser">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leUser">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPass">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePass">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>361</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnSelectFile">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>71</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Datei wählen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSelectFile">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblFileFormat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Das akzeptierte Audioformat für Mixe ist: MP3, 192kb/s</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblNameConvention">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Der Dateiname setzt sich aus dem Namen und Datum der Sendung zusammen</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>219</y>
       <width>253</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Datum der Sendung:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>d/M/yy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2017</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="leShowName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>125</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblShowName">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>180</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Name der Sendung:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnUpload">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hochladen</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>106</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QRadioButton" name="rbtn2hShow">
       <property name="text">
        <string>2h</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="rbtn3hShow">
       <property name="text">
        <string>3h</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rbtn1hShow">
       <property name="text">
        <string>1h</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblRbtns">
       <property name="text">
        <string>Länge des Mixes:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Der Mix sollte 30-60 Sekunden kürzer als die Sendung sein.
Mix für 1h = 0:59:00 - 0:59:30</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leHost</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>leSelectFile</tabstop>
  <tabstop>rbtn1hShow</tabstop>
  <tabstop>rbtn2hShow</tabstop>
  <tabstop>rbtn3hShow</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>btnUpload</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>selectFile()</slot>
  <slot>uploadMix()</slot>
 </slots>
</ui>
